app/src/androidTest/java/com/github/onlynotesswent/ui/overview/FolderContentTest.kt

```diff
@@ -0,0 +1,157 @@
+package com.github.onlynotesswent.ui.overview
+
+import android.graphics.Bitmap
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import com.github.onlynotesswent.model.folder.Folder
+import com.github.onlynotesswent.model.folder.FolderRepository
+import com.github.onlynotesswent.model.folder.FolderViewModel
+import com.github.onlynotesswent.model.note.Note
+import com.github.onlynotesswent.model.note.NoteRepository
+import com.github.onlynotesswent.model.note.NoteViewModel
+import com.github.onlynotesswent.model.users.User
+import com.github.onlynotesswent.model.users.UserRepository
+import com.github.onlynotesswent.model.users.UserViewModel
+import com.github.onlynotesswent.ui.navigation.NavigationActions
+import com.github.onlynotesswent.ui.navigation.Screen
+import com.github.onlynotesswent.ui.navigation.TopLevelDestinations
+import com.google.firebase.Timestamp
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.mockito.Mockito.`when`
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+
+class FolderContentTest {
+  private lateinit var userRepository: UserRepository
```

Comment:
Try Using @Mock annotations, then initialize with `MockitoAnnotations.openMocks(this)` for easier initialisation